samcli/lib/utils/subprocess_utils.py

```diff
@@ -92,10 +96,18 @@ def _print_loading_pattern():
                     # we read from subprocess stdout to avoid the deadlock process.wait function
                     # for more detail check this python bug https://bugs.python.org/issue1256
                     for line in process.stdout:
-                        decoded_line = _check_and_process_bytes(line)
+                        is_error = (
+                            not IS_NOT_WINDOWS
```

Comment:
nit: Can we maybe flip the constant to clean up this condition? I think it would make it easier to read since this double negative reads a little awkward. 

`IS_WINDOWS` instead of `IS_NOT_WINDOWS`